src/cascadia/TerminalSettingsEditor/ColorSchemeViewModel.cpp

```diff
@@ -49,6 +50,16 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _Name;
     }
 
+    bool ColorSchemeViewModel::IsDefaultScheme()
+    {
+        return _Name == _settings.ProfileDefaults().DefaultAppearance().ColorSchemeName();

```

Comment:
_clever_. renaming a scheme updates all profiles using it, so this will be true after a rename too. very clever.